hw1/task2/bash.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+
+def wc(first: str, *args: str):
+    def wc_for_one(filepath: str):
+        size = os.path.getsize(filepath)
+        f = open(filepath)
+        words = 0
+        lines = 0
+        for line in f:
+            lines += 1
+            words += len(line.split())
+        f.close()
+        return {"newlines": lines, "words": words, "bytes": size}
+
+    acc = {"newlines": 0, "words": 0, "bytes": 0}
+    files_not_found = []
+    for filename in [first, *args]:
+        try:
+            file_wc = wc_for_one(filename)
+        except FileNotFoundError:
+            files_not_found.append(filename)
+            continue
+        print(str(file_wc) + "\t" + filename)
+        acc = {k: acc.get(k, 0) + file_wc.get(k, 0) for k in acc}
+    print(str(acc), "\ttotal")
+    if files_not_found:
+        print("Files not found: ", end="")
+        for file in files_not_found:
+            print(file, end=", ")
+
+
+def nl(filepath: str):
+    try:
+        file = open(filepath)
+    except FileNotFoundError:
+        print("No such file: ", filepath)
+        return
+    count = 1
+    for line in file:
+        if line == "\n":
+            print()
+        else:
+            print(str(count) + " " + line, end="")
+            count += 1
+    file.close()
+
+
+def head(filename: str, n=10):
```

Comment:
Even though the second argument has a default value, it can still be typed. Please, google how to do this and add a typing of `int` there.

Reply:
In `tail` also.